.github/workflows/notifier.yml

```diff
@@ -37,14 +37,14 @@ jobs:
         run: |
           # Add community label
           curl -X POST \
-            -H "Authorization: Bearer ${{ secrets.GH_ORG_Members_PAT }}" \
+            -H "Authorization: token $GH_TOKEN" \
```

Comment:
Is `token` a valid [authentication scheme](https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication#authentication_schemes) or was this change perhaps accidental?

Reply:
it's valid, this passed trough repo settings secrets 

see https://github.com/weaviate/weaviate/pull/7039/files#diff-dbfa576d113713d39d0728614267985772b57757c5200e16e1297814349a6ba2R36

Reply:
actually GH supports both https://docs.github.com/en/rest/authentication/authenticating-to-the-rest-api?apiVersion=2022-11-28#authenticating-with-a-token 

<img width="729" alt="Screenshot 2025-01-24 at 16 45 04" src="https://github.com/user-attachments/assets/34067923-eded-473e-ab6f-7b18e600e17b" />


however i am gonna make it `Bearer` to not confuse anyone